Jellyfin.Server/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -107,6 +110,7 @@ internal static string CheckBadChars(string name)
                 switch (crlf)
                 {
                     case 0:
+                    {
```

Comment:
why add braces here? they seem to be excessive for me

Reply:
Pretty sure the the analyzer wanted it.